### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by rega1237**
In this branch @devMahmoud  and I Check 
- [ ] Page titles : Change the title https://github.com/rega1237/rega1237.github.io/blob/49610bcedb5331a669cb5cb126e8b75797802a75/index.html#L11

- [ ] Image text alternatives : Change Alt text on the social icon images
https://github.com/rega1237/rega1237.github.io/blob/49610bcedb5331a669cb5cb126e8b75797802a75/index.html#L45
https://github.com/rega1237/rega1237.github.io/blob/49610bcedb5331a669cb5cb126e8b75797802a75/index.html#L48
https://github.com/rega1237/rega1237.github.io/blob/49610bcedb5331a669cb5cb126e8b75797802a75/index.html#L51
https://github.com/rega1237/rega1237.github.io/blob/49610bcedb5331a669cb5cb126e8b75797802a75/index.html#L54
https://github.com/rega1237/rega1237.github.io/blob/49610bcedb5331a669cb5cb126e8b75797802a75/index.html#L57
https://github.com/rega1237/rega1237.github.io/blob/49610bcedb5331a669cb5cb126e8b75797802a75/index.html#L216
https://github.com/rega1237/rega1237.github.io/blob/49610bcedb5331a669cb5cb126e8b75797802a75/index.html#L231
https://github.com/rega1237/rega1237.github.io/blob/49610bcedb5331a669cb5cb126e8b75797802a75/index.html#L244
https://github.com/rega1237/rega1237.github.io/blob/49610bcedb5331a669cb5cb126e8b75797802a75/index.html#L307
https://github.com/rega1237/rega1237.github.io/blob/49610bcedb5331a669cb5cb126e8b75797802a75/index.html#L311
https://github.com/rega1237/rega1237.github.io/blob/49610bcedb5331a669cb5cb126e8b75797802a75/index.html#L315
https://github.com/rega1237/rega1237.github.io/blob/49610bcedb5331a669cb5cb126e8b75797802a75/index.html#L319
https://github.com/rega1237/rega1237.github.io/blob/49610bcedb5331a669cb5cb126e8b75797802a75/index.html#L323

- [ ] Text headings : Change some h tags in the file
https://github.com/rega1237/rega1237.github.io/blob/49610bcedb5331a669cb5cb126e8b75797802a75/index.html#L78
https://github.com/rega1237/rega1237.github.io/blob/49610bcedb5331a669cb5cb126e8b75797802a75/index.html#L99
https://github.com/rega1237/rega1237.github.io/blob/49610bcedb5331a669cb5cb126e8b75797802a75/index.html#L120
https://github.com/rega1237/rega1237.github.io/blob/49610bcedb5331a669cb5cb126e8b75797802a75/index.html#L141
https://github.com/rega1237/rega1237.github.io/blob/49610bcedb5331a669cb5cb126e8b75797802a75/index.html#L162
https://github.com/rega1237/rega1237.github.io/blob/49610bcedb5331a669cb5cb126e8b75797802a75/index.html#L183
https://github.com/rega1237/rega1237.github.io/blob/49610bcedb5331a669cb5cb126e8b75797802a75/index.html#L260
https://github.com/rega1237/rega1237.github.io/blob/49610bcedb5331a669cb5cb126e8b75797802a75/index.html#L263

- [ ] Color contrast :  no issues found
- [ ] Resize :  no issues found
- [ ] Interaction :  no issues found
- [ ] Moving content :  no issues found
- [ ] Multimedia :  no issues found
- [ ] The basic structure of the page :  no issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <style>
       @import url("https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap");
     </style>
-    <title>Portafolio</title>
+    <title>Rafael Portafolio home page</title>
```
Comment (micheaol) [inline index.html]: Kindly update your page titles.

It adequately and briefly describes the content of the page. For example:
### Poor titles:
Microverse project | Your portfolio.
Portifolio project


### Better page titles:
Mark's portifolio
Mark full-stack webdeveloper

File: index.html [hunk 2]
```diff
@@ -257,10 +257,10 @@ <h3>Skills</h3>
...
       <section class="form-contact">
-        <h3 class="text-contact">
+        <p class="text-contact">
           I'm always interested in hearing about new projects, so if you'd like
           to chat please get in touch.
-        </h3>
+        </p>
         <form
           id="template-form"
           action="https://formspree.io/f/mbjwkvzr"
```
Comment (micheaol) [inline index.html:266]: Please ensure that every form control has a label associated with it using 'label', 'for', and 'id'.

File: styles.css [hunk 3]
```diff
@@ -100,7 +100,7 @@ main {
   background-color: #ebf0ee;
 }
```
Comment (micheaol) [inline styles.css:102]: Please ensure that the page has a minimum contrast ratio of at least 4.5:1 for normal-size text. Below is the test result, some of the tests are failing. Kindly look into this.

https://www.aditus.io/button-contrast-checker/rega1237-github-io-2022-03-29-at-21-15-53-167?failing

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @rega1237 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by keroloussamy — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap

## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**